tests/Storage.test.ts

```diff
@@ -12,9 +12,15 @@
 *******************************************************************************/
 
 import * as assert from "assert";
-import { Block, Endian, Hash, Height, JSBI, PreImageInfo, SodiumHelper } from "boa-sdk-ts";
+import {Block, BlockHeader, Endian, Hash, Height, JSBI, PreImageInfo, SodiumHelper} from "boa-sdk-ts";
```

Comment:
Isn't the coding style automated? Please check it.

Reply:
I Applied it.